### PR TITLE
Add Register Copilot Capability procedure for specific app id to Copilot Test Library

### PR DESCRIPTION
**Pull request by tomasevicst**
#### Summary
To be able to run SLS tests in pipeline, we need overwritten procedure "Register Copilot Capability", that will allow us to register copilot capability for specifi app id.

#### Work Item(s) 
[AB#543582](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/543582) Add Register Copilot Capability for specific app id  procedure to "Copilot Test Library"

### CODE REVIEW

File: src/System Application/Test Library/AI/src/CopilotTestLibrary.Codeunit.al [hunk 1]
```diff
@@ -29,6 +29,18 @@ codeunit 132932 "Copilot Test Library"
         CopilotCapability.RegisterCapability(Capability, '');
     end;
 
+    procedure RegisterCopilotCapabilityForSpecificAppId(Capability: Enum "Copilot Capability"; AppId: Guid)
```
Comment (darjoo) [inline src/System Application/Test Library/AI/src/CopilotTestLibrary.Codeunit.al]:
```suggestion
    procedure RegisterCopilotCapabilityWithAppId(Capability: Enum "Copilot Capability"; AppId: Guid)
```

File: src/System Application/Test Library/AI/src/CopilotTestLibrary.Codeunit.al [hunk 2]
```diff
@@ -29,6 +29,18 @@ codeunit 132932 "Copilot Test Library"
...
+    var
+        CopilotSettings: Record "Copilot Settings";
+    begin
+        CopilotCapability.RegisterCapability(Capability, '');
```
Comment (darjoo) [inline src/System Application/Test Library/AI/src/CopilotTestLibrary.Codeunit.al]:
```suggestion
        RegisterCopilotCapability(Capability);
```

Use as overload. First overload should be changed but that's fine for now.

File: src/System Application/Test Library/AI/src/CopilotTestLibrary.Codeunit.al (hunk 1 above)
Comment (tomasevicst) [reply src/System Application/Test Library/AI/src/CopilotTestLibrary.Codeunit.al]: Fixed

File: src/System Application/Test Library/AI/src/CopilotTestLibrary.Codeunit.al (hunk 2 above)
Comment (tomasevicst) [reply src/System Application/Test Library/AI/src/CopilotTestLibrary.Codeunit.al]: Fixed

**Review by darjoo — APPROVED** [review]

**Review by t-prda — APPROVED** [review]

**Pull request merged**